docs/build/getting-started/authenticating-with-lit.md

```diff
@@ -1 +1,43 @@
-# Authenticating with Lit
\ No newline at end of file
+# Authenticating with Lit
+After connecting, sending requests to the Lit network requires authentication. This authentication is necessary to ensure that the Lit network can identify your session and only grant you access to the resources you're authorized to access.
+
+## Session Signatures
+Lit uses Session Signatures to authenticate your session. A detailed description of Session Signatures can be found [here](../session-sigs/overview.md).
+
+### Generating Session Signatures
+The Lit SDK has multiple methods for generating Session Signatures:
+
+- [`getSessionSigs`](../session-sigs/generating-a-session/using-auth-sig.md)
```

Comment:
maybe we could add ~1 sentence briefly explaining what each method does / when it should be used? Just a brief intro to guide folks on where they should redirect to based on their use case